include/zephyr/drivers/video-controls.h

```diff
@@ -107,18 +227,307 @@ enum video_power_line_frequency {
+
+/** Determines the ISO equivalent of an image sensor indicating the sensor's sensitivity to light.
+ * The numbers are expressed in linear scale. Applications should interpret the values as standard
+ * ISO values multiplied by 1000, e.g. control value 800 stands for ISO 0.8. Drivers will usually
+ * support only a subset of standard ISO values.
+ * Setting this control when @ref VIDEO_CID_ISO_SENSITIVITY_AUTO is not
+ * @ref VIDEO_ISO_SENSITIVITY_MANUAL is undefined.
+ * Drivers should ignore such requests.
+ */
+#define VIDEO_CID_ISO_SENSITIVITY (VIDEO_CID_CAMERA_CLASS_BASE + 23)
+
+/** Enables or disables automatic ISO sensitivity adjustments. */
+#define VIDEO_CID_ISO_SENSITIVITY_AUTO (VIDEO_CID_CAMERA_CLASS_BASE + 24)
+enum video_iso_sensitivity_auto {
+	VIDEO_ISO_SENSITIVITY_MANUAL = 0,
+	VIDEO_ISO_SENSITIVITY_AUTO = 1,
+};
+
+/** Determines how the camera measures the amount of light available for the frame exposure. */
+#define VIDEO_CID_EXPOSURE_METERING (VIDEO_CID_CAMERA_CLASS_BASE + 25)
+enum video_exposure_metering {
+	VIDEO_EXPOSURE_METERING_AVERAGE = 0,
+	VIDEO_EXPOSURE_METERING_CENTER_WEIGHTED = 1,
+	VIDEO_EXPOSURE_METERING_SPOT = 2,
+	VIDEO_EXPOSURE_METERING_MATRIX = 3,
+};
+
+/** This control selects scene programs optimized for common shooting scenes.
+ * Within these modes the camera determines best exposure, aperture, focusing, light metering,
+ * white balance and equivalent sensitivity. The controls of those parameters are influenced by
+ * the scene mode control. An exact behavior in each mode is subject to the camera specification.
+ * When the scene mode feature is not used, this control should be set to
+ * @ref VIDEO_SCENE_MODE_NONE to make sure the other possibly related controls are accessible.
+ */
+#define VIDEO_CID_SCENE_MODE (VIDEO_CID_CAMERA_CLASS_BASE + 26)
+enum video_scene_mode {
+	VIDEO_SCENE_MODE_NONE = 0,
+	VIDEO_SCENE_MODE_BACKLIGHT = 1,
+	VIDEO_SCENE_MODE_BEACH_SNOW = 2,
+	VIDEO_SCENE_MODE_CANDLE_LIGHT = 3,
+	VIDEO_SCENE_MODE_DAWN_DUSK = 4,
+	VIDEO_SCENE_MODE_FALL_COLORS = 5,
+	VIDEO_SCENE_MODE_FIREWORKS = 6,
+	VIDEO_SCENE_MODE_LANDSCAPE = 7,
+	VIDEO_SCENE_MODE_NIGHT = 8,
+	VIDEO_SCENE_MODE_PARTY_INDOOR = 9,
+	VIDEO_SCENE_MODE_PORTRAIT = 10,
+	VIDEO_SCENE_MODE_SPORTS = 11,
+	VIDEO_SCENE_MODE_SUNSET = 12,
+	VIDEO_SCENE_MODE_TEXT = 13,
+};
+
+/** This control locks or unlocks the automatic focus, exposure and white balance.
+ * The automatic adjustments can be paused independently by setting their lock bit to 1.
+ * The camera then retains the settings until the lock bit is cleared.
+ * When a given algorithm is not enabled, drivers return no error.
+ * An example might be an application setting bit @ref VIDEO_LOCK_WHITE_BALANCE when the
+ * @ref VIDEO_CID_AUTO_WHITE_BALANCE control is off (set to 0).
+ * The value of this control may be changed by exposure, white balance or focus controls.
+ */
+#define VIDEO_CID_3A_LOCK (VIDEO_CID_CAMERA_CLASS_BASE + 27)
```

Comment:
REMOVE? Used in only two Linux driver:
`drivers/media/i2c/s5c73m3/s5c73m3-ctrls.c`
`drivers/media/platform/samsung/exynos4-is/fimc-isp.c`